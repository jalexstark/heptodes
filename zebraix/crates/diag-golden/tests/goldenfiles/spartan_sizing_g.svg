<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400pt" height="300pt" viewBox="0 0 400 300">
<defs>
<g>
<g id="glyph-0-0">
<path d="M 0.375 -2.140625 L 0.375 -2.875 L 2.640625 -2.875 L 2.640625 -2.140625 Z M 0.375 -2.140625 "/>
</g>
<g id="glyph-0-1">
<path d="M 4.859375 0 L 0.453125 0 L 0.453125 -0.6875 L 2.203125 -2.453125 C 2.535156 -2.785156 2.816406 -3.082031 3.046875 -3.34375 C 3.273438 -3.601562 3.445312 -3.859375 3.5625 -4.109375 C 3.6875 -4.367188 3.75 -4.644531 3.75 -4.9375 C 3.75 -5.3125 3.640625 -5.59375 3.421875 -5.78125 C 3.203125 -5.96875 2.914062 -6.0625 2.5625 -6.0625 C 2.238281 -6.0625 1.953125 -6.003906 1.703125 -5.890625 C 1.460938 -5.785156 1.21875 -5.628906 0.96875 -5.421875 L 0.515625 -5.984375 C 0.785156 -6.203125 1.085938 -6.382812 1.421875 -6.53125 C 1.765625 -6.6875 2.144531 -6.765625 2.5625 -6.765625 C 3.1875 -6.765625 3.679688 -6.609375 4.046875 -6.296875 C 4.410156 -5.984375 4.59375 -5.546875 4.59375 -4.984375 C 4.59375 -4.640625 4.519531 -4.3125 4.375 -4 C 4.226562 -3.695312 4.023438 -3.394531 3.765625 -3.09375 C 3.515625 -2.800781 3.21875 -2.488281 2.875 -2.15625 L 1.484375 -0.78125 L 1.484375 -0.75 L 4.859375 -0.75 Z M 4.859375 0 "/>
</g>
<g id="glyph-0-2">
<path d="M 4.890625 -3.34375 C 4.890625 -2.625 4.8125 -2.007812 4.65625 -1.5 C 4.507812 -0.988281 4.273438 -0.59375 3.953125 -0.3125 C 3.628906 -0.0390625 3.195312 0.09375 2.65625 0.09375 C 1.90625 0.09375 1.347656 -0.207031 0.984375 -0.8125 C 0.628906 -1.425781 0.453125 -2.269531 0.453125 -3.34375 C 0.453125 -4.0625 0.523438 -4.675781 0.671875 -5.1875 C 0.816406 -5.707031 1.050781 -6.101562 1.375 -6.375 C 1.707031 -6.644531 2.132812 -6.78125 2.65625 -6.78125 C 3.40625 -6.78125 3.960938 -6.476562 4.328125 -5.875 C 4.703125 -5.269531 4.890625 -4.425781 4.890625 -3.34375 Z M 1.28125 -3.34375 C 1.28125 -2.425781 1.382812 -1.738281 1.59375 -1.28125 C 1.800781 -0.832031 2.15625 -0.609375 2.65625 -0.609375 C 3.15625 -0.609375 3.515625 -0.832031 3.734375 -1.28125 C 3.953125 -1.738281 4.0625 -2.425781 4.0625 -3.34375 C 4.0625 -4.25 3.953125 -4.929688 3.734375 -5.390625 C 3.515625 -5.847656 3.15625 -6.078125 2.65625 -6.078125 C 2.15625 -6.078125 1.800781 -5.847656 1.59375 -5.390625 C 1.382812 -4.929688 1.28125 -4.25 1.28125 -3.34375 Z M 1.28125 -3.34375 "/>
</g>
<g id="glyph-0-3">
<path d="M 3.3125 0 L 2.515625 0 L 2.515625 -4.65625 C 2.515625 -4.925781 2.515625 -5.140625 2.515625 -5.296875 C 2.523438 -5.460938 2.535156 -5.632812 2.546875 -5.8125 C 2.453125 -5.707031 2.363281 -5.617188 2.28125 -5.546875 C 2.195312 -5.484375 2.09375 -5.398438 1.96875 -5.296875 L 1.265625 -4.71875 L 0.828125 -5.265625 L 2.640625 -6.671875 L 3.3125 -6.671875 Z M 3.3125 0 "/>
</g>
<g id="glyph-0-4">
<path d="M 0.671875 -0.5 C 0.671875 -0.726562 0.726562 -0.890625 0.84375 -0.984375 C 0.957031 -1.078125 1.09375 -1.125 1.25 -1.125 C 1.40625 -1.125 1.539062 -1.078125 1.65625 -0.984375 C 1.769531 -0.890625 1.828125 -0.726562 1.828125 -0.5 C 1.828125 -0.28125 1.769531 -0.117188 1.65625 -0.015625 C 1.539062 0.078125 1.40625 0.125 1.25 0.125 C 1.09375 0.125 0.957031 0.078125 0.84375 -0.015625 C 0.726562 -0.117188 0.671875 -0.28125 0.671875 -0.5 Z M 0.671875 -0.5 "/>
</g>
<g id="glyph-0-5">
<path d="M 2.65625 -6.765625 C 3.25 -6.765625 3.726562 -6.625 4.09375 -6.34375 C 4.46875 -6.070312 4.65625 -5.679688 4.65625 -5.171875 C 4.65625 -4.898438 4.597656 -4.664062 4.484375 -4.46875 C 4.367188 -4.269531 4.210938 -4.09375 4.015625 -3.9375 C 3.828125 -3.78125 3.617188 -3.644531 3.390625 -3.53125 C 3.660156 -3.394531 3.90625 -3.242188 4.125 -3.078125 C 4.351562 -2.921875 4.535156 -2.734375 4.671875 -2.515625 C 4.804688 -2.296875 4.875 -2.035156 4.875 -1.734375 C 4.875 -1.171875 4.675781 -0.722656 4.28125 -0.390625 C 3.882812 -0.0664062 3.351562 0.09375 2.6875 0.09375 C 1.96875 0.09375 1.414062 -0.0625 1.03125 -0.375 C 0.644531 -0.6875 0.453125 -1.128906 0.453125 -1.703125 C 0.453125 -2.003906 0.515625 -2.265625 0.640625 -2.484375 C 0.773438 -2.710938 0.945312 -2.90625 1.15625 -3.0625 C 1.363281 -3.226562 1.59375 -3.367188 1.84375 -3.484375 C 1.519531 -3.671875 1.242188 -3.894531 1.015625 -4.15625 C 0.785156 -4.425781 0.671875 -4.765625 0.671875 -5.171875 C 0.671875 -5.515625 0.757812 -5.800781 0.9375 -6.03125 C 1.113281 -6.269531 1.347656 -6.453125 1.640625 -6.578125 C 1.941406 -6.703125 2.28125 -6.765625 2.65625 -6.765625 Z M 2.65625 -6.09375 C 2.3125 -6.09375 2.023438 -6.007812 1.796875 -5.84375 C 1.578125 -5.6875 1.46875 -5.453125 1.46875 -5.140625 C 1.46875 -4.910156 1.519531 -4.71875 1.625 -4.5625 C 1.738281 -4.40625 1.890625 -4.269531 2.078125 -4.15625 C 2.265625 -4.050781 2.472656 -3.945312 2.703125 -3.84375 C 3.023438 -3.988281 3.296875 -4.15625 3.515625 -4.34375 C 3.734375 -4.539062 3.84375 -4.804688 3.84375 -5.140625 C 3.84375 -5.453125 3.734375 -5.6875 3.515625 -5.84375 C 3.304688 -6.007812 3.019531 -6.09375 2.65625 -6.09375 Z M 1.265625 -1.6875 C 1.265625 -1.363281 1.378906 -1.09375 1.609375 -0.875 C 1.835938 -0.664062 2.191406 -0.5625 2.671875 -0.5625 C 3.128906 -0.5625 3.476562 -0.664062 3.71875 -0.875 C 3.957031 -1.09375 4.078125 -1.375 4.078125 -1.71875 C 4.078125 -2.039062 3.945312 -2.304688 3.6875 -2.515625 C 3.4375 -2.734375 3.097656 -2.925781 2.671875 -3.09375 L 2.515625 -3.15625 C 2.109375 -2.976562 1.796875 -2.773438 1.578125 -2.546875 C 1.367188 -2.316406 1.265625 -2.03125 1.265625 -1.6875 Z M 1.265625 -1.6875 "/>
</g>
</g>
</defs>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 66.070312 283.929688 L 66.070312 16.070312 M 333.929688 283.929688 L 333.929688 16.070312 M 66.070312 283.929688 L 333.929688 283.929688 M 66.070312 16.070312 L 333.929688 16.070312 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 66.070312 150 L 333.929688 150 M 66.070312 190.179688 L 333.929688 190.179688 M 66.070312 230.355469 L 333.929688 230.355469 M 66.070312 270.535156 L 333.929688 270.535156 M 66.070312 109.820312 L 333.929688 109.820312 M 66.070312 69.644531 L 333.929688 69.644531 M 66.070312 29.464844 L 333.929688 29.464844 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 200 283.929688 L 200 16.070312 M 173.214844 283.929688 L 173.214844 16.070312 M 146.429688 283.929688 L 146.429688 16.070312 M 119.644531 283.929688 L 119.644531 16.070312 M 92.855469 283.929688 L 92.855469 16.070312 M 226.785156 283.929688 L 226.785156 16.070312 M 253.570312 283.929688 L 253.570312 16.070312 M 280.355469 283.929688 L 280.355469 16.070312 M 307.144531 283.929688 L 307.144531 16.070312 "/>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-0" x="62.070312" y="295.791992"/>
<use xlink:href="#glyph-0-1" x="65.070312" y="295.791992"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-2" x="197.5" y="295.791992"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-1" x="331.429688" y="295.791992"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-0" x="45.394531" y="273.311523"/>
<use xlink:href="#glyph-0-3" x="48.394531" y="273.311523"/>
<use xlink:href="#glyph-0-4" x="53.394531" y="273.311523"/>
<use xlink:href="#glyph-0-5" x="56.394531" y="273.311523"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-2" x="48.394531" y="152.776367"/>
<use xlink:href="#glyph-0-4" x="53.394531" y="152.776367"/>
<use xlink:href="#glyph-0-2" x="56.394531" y="152.776367"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-3" x="48.394531" y="32.241211"/>
<use xlink:href="#glyph-0-4" x="53.394531" y="32.241211"/>
<use xlink:href="#glyph-0-5" x="56.394531" y="32.241211"/>
</g>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 133.035156 216.964844 L 133.035156 83.035156 M 266.964844 216.964844 L 266.964844 83.035156 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 133.035156 83.035156 L 266.964844 216.964844 M 266.964844 83.035156 L 133.035156 216.964844 "/>
</svg>
