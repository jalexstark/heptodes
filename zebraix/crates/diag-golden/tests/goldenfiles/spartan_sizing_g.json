{
  "sizing_scheme": "SquareCenter",
  "axes_range": [
    2.0
  ],
  "padding": [
    0.06
  ],
  "drawables": [
    {
      "drawable": {
        "Lines": {
          "start": [
            [
              -2.0,
              -2.0
            ],
            [
              2.0,
              -2.0
            ],
            [
              -2.0,
              -2.0
            ],
            [
              -2.0,
              2.0
            ]
          ],
          "end": [
            [
              -2.0,
              2.0
            ],
            [
              2.0,
              2.0
            ],
            [
              2.0,
              -2.0
            ],
            [
              2.0,
              2.0
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "start": [
            [
              -2.0,
              0.0
            ]
          ],
          "end": [
            [
              2.0,
              0.0
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ],
            [
              -0.0,
              -0.6
            ],
            [
              -0.0,
              -1.2
            ],
            [
              -0.0,
              -1.7999999999999998
            ],
            [
              0.0,
              0.6
            ],
            [
              0.0,
              1.2
            ],
            [
              0.0,
              1.7999999999999998
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "start": [
            [
              0.0,
              -2.0
            ]
          ],
          "end": [
            [
              0.0,
              2.0
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ],
            [
              -0.4,
              -0.0
            ],
            [
              -0.8,
              -0.0
            ],
            [
              -1.2000000000000002,
              -0.0
            ],
            [
              -1.6,
              -0.0
            ],
            [
              0.4,
              0.0
            ],
            [
              0.8,
              0.0
            ],
            [
              1.2000000000000002,
              0.0
            ],
            [
              1.6,
              0.0
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Text": {
          "size_choice": "Small",
          "offset_choice": "Both",
          "anchor_vertical": "Top",
          "texts": [
            {
              "content": "-2",
              "location": [
                -2.0,
                -2.0
              ]
            },
            {
              "content": "0",
              "location": [
                0.0,
                -2.0
              ]
            },
            {
              "content": "2",
              "location": [
                2.0,
                -2.0
              ]
            }
          ]
        }
      }
    },
    {
      "drawable": {
        "Text": {
          "size_choice": "Small",
          "offset_choice": "Both",
          "anchor_horizontal": "Right",
          "texts": [
            {
              "content": "-1.8",
              "location": [
                -2.0,
                -1.7999999999999998
              ]
            },
            {
              "content": "0.0",
              "location": [
                -2.0,
                0.0
              ]
            },
            {
              "content": "1.8",
              "location": [
                -2.0,
                1.7999999999999998
              ]
            }
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "start": [
            [
              -1.0,
              -1.0
            ]
          ],
          "end": [
            [
              -1.0,
              1.0
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ],
            [
              2.0,
              0.0
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "start": [
            [
              -1.0,
              1.0
            ],
            [
              1.0,
              1.0
            ]
          ],
          "end": [
            [
              1.0,
              -1.0
            ],
            [
              -1.0,
              -1.0
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ]
          ]
        }
      }
    }
  ]
}
